dist/gen/js/genTypes.js

```diff
@@ -134,7 +134,15 @@ export interface OperationSecurity {
 export interface OperationParamGroups {
   header?: {[key: string]: string}${support_1.ST}
   path?: {[key: string]: string|number|boolean}${support_1.ST}
-  query?: {[key: string]: string|string[]|number|boolean}${support_1.ST}
+  query?: {[key: string]:      
+      | string
+      | string[]
+      | number
+      | number[]
```

Comment:
Das waren die Fehler in den API definitionen. Das script ist hier restriktiver in den query typen als unsere swagger definition. Ich habe jetzt nur mit aufgenommen wo ich bisher einen Fehler bekommen habe also ggf. wächst hier die List. Mann könnte über any 🫣 nachdenken. Da die Parameter der funktionen ja schon getyped sind.